bin/init_dmftpr

```diff
@@ -0,0 +1,118 @@
+        for atom in range(num_atoms):
+            while True: # input choice of spherical harmonics for atom
+                sph_harm=input("What flavor of spherical harmonics do you want to use for ATOM {} ({})? (cubic/complex/fromfile)\n".format(atom+1, species[atom]))
+                if sph_harm in ["cubic", "complex", "fromfile"]:
+                    out.write(sph_harm+"\n")
+                    if sph_harm == "fromfile":
+                        filename=input("name of file defining the basis?\n")
+                        if len(filename) < 25: # name of file must be less than 25 characters
+                            out.write(filename)
+                        else:
+                            print("{} is too long!".format(filename))
+                            rename=input("Rename the file to: \n")
+                            if os.path.isfile(filename):
+                                os.rename(filename, rename)
+                                out.write(rename+"\n")
+                            else:
+                                print("{} could not be found in the current directory!".format(filename))
+                                sys.exit(1)
+                    break
+
+                else:
+                    print("Did not recognize that input. Try again.")
+
+            corr=input("Do you want to treat ATOM {} ({}) as correlated (y/n)?\n".format(atom+1, species[atom]))
+            if corr == "y":
+                proj=input("Specify the correlated orbital? (d,f)\n")
+                while True:
+                    non_corr=input("projectors for non-correlated orbitals? (type h for help)\n")
+                    if non_corr == "h":
+                        print("indicate orbital projectors using (s, p, d, or f). For multiple, combine them (sp, pd, spd, etc.)")
+                    elif len(non_corr) > 0 and proj in non_corr:
+                        print("Error: User can not choose orbital {} as both correlated and uncorrelated!".format(proj))
+                    else:
+                        projectors=array([0, 0, 0, 0])
+                        projectors += array(corr_orbitals[proj])
+                        if len(non_corr) > 0:
+                            for p in non_corr: projectors += array(orbitals[p])
+                        out.write(" ".join(list(map(str, projectors)))+"\n")
+                        break
+                if proj == "d":
+                    irrep=input("Split this orbital into it's irreps? (t2g/eg/n)\n")
+                    to_write=""
+                    if irrep == "t2g":
+                        to_write += "0 0 2 0\n01\n"
+                    elif irrep == "eg":
+                        to_write += "0 0 2 0\n10\n"
+                    else:
+                        to_write += "0 0 0 0\n"
+
+                soc=input("Do you want to include soc? (y/n)\n")
+                if soc == "y":
+                    if irrep == "t2g" or irrep == "eg":
```

Comment:
```suggestion
                    if proj=="d" and ( irrep == "t2g" or irrep == "eg" ):
```
This whole block only works for the d-shell (irrep and to_write is not defined otherwise)